### PR TITLE
Generate true sentences

### PR DESCRIPTION
**Pull request by GwydionJon**
This adds the ability to use the actual sentences instead of the paragraphs.

### CODE REVIEW

File: notebooks/DemoNotebook_statistics.ipynb [hunk 1]
```diff
@@ -56,7 +56,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_dict = mn.InputOutput.get_input_dir(\"Test_Data/XMI_11\")"
+    "data_dict = mn.InputOutput.get_input_dir(\"../../moralization_data/Test_Data/XMI_11\")"
```
Comment (lkeegan) [inline notebooks/DemoNotebook_statistics.ipynb:59]:
```suggestion
    "data_dict = mn.InputOutput.get_input_dir(\"Test_Data/XMI_11\")"
```

File: moralization/analyse.py [hunk 2]
```diff
@@ -71,14 +73,55 @@ def sort_spans(cas: object, ts: object) -> defaultdict:
...
+    text = cas.sofa_string
+    sentences = [sentence for sentence in nlp(text).sents]
+    # create new typesystem:
+    try:
+        True_sentence_type = ts.create_type(name="moralization.TrueSentence")
+    except ValueError:
+        True_sentence_type = ts.get_type("moralization.TrueSentence")
+
+    for sentence in sentences:
+        if (sentence.end - sentence.start) > 3:
```
Comment (lkeegan) [inline moralization/analyse.py:87]: could `3` be a variable with a name that suggests something about what it is, e.g. `min_sentence_length`?

File: moralization/analyse.py [hunk 3]
```diff
@@ -71,14 +73,55 @@ def sort_spans(cas: object, ts: object) -> defaultdict:
...
+            # spacy returns the token ids, not the character ids.
+            # this is in missmatch with cassis.
+            # to convert from token to character I search through the doc with a regex to find the correct ids.
+            # the replace is to avoid regex errors in regards to brackets
```
Comment (lkeegan) [inline moralization/analyse.py:91]: I'm confused by this - does spacy not allow you to see the location of a token, e.g. `token.idx`?

**Review by lkeegan — COMMENTED** [review]
A general comment - this would really benefit from a few tests, as well as the usual reasons it would be helpfult to see simple examples of input -> output for a lot of these functions

File: moralization/analyse.py (hunk 2 above)
Comment (GwydionJon) [reply moralization/analyse.py:87]: done under the name `minimum_tokens_per_sentence`

File: moralization/analyse.py (hunk 3 above)
Comment (GwydionJon) [reply moralization/analyse.py:91]: My mistake, i first only found the `token.start`and `token.end` variables.
but spacy also has `token.start_char`which does exactly what I needed.

**Pull request closed without merging**